src/AwkwardArray.jl

```diff
@@ -9,5 +9,9 @@ include("./tables.jl")
 include("./AwkwardPythonCallExt.jl")
 using .AwkwardPythonCallExt: convert
 
+Base.eltype(::RecordArray{FIELDS,CONTENTS,BEHAVIOR}) where {FIELDS,CONTENTS,BEHAVIOR} = Record{FIELDS,CONTENTS,BEHAVIOR}
+Base.eltype(::Record{FIELDS,CONTENTS,BEHAVIOR}) where {FIELDS,CONTENTS,BEHAVIOR} = CONTENTS
+Base.eltype(::TupleArray{CONTENTS,BEHAVIOR}) where {CONTENTS,BEHAVIOR} = Tuple{CONTENTS,BEHAVIOR}
+Base.eltype(::UnionArray{TAGS,INDEX,CONTENTS}) where {TAGS,INDEX,CONTENTS} = CONTENTS
```

Comment:
Is `CONTENTS` really a Julia `Union`? The element type of something extracted from a `UnionArray` should be a _union_ of all of its content types. I think the `CONTENTS` parameter is some kind of list (Julia `Tuple`?).